packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts

```diff
@@ -279,37 +299,38 @@ export const useNotifiSubscribe: () => Readonly<{
           );
         }
 
+        const alert = await client.createAlert({
+          emailAddress: finalEmail,
+          filterId: filter?.id ?? '',
+          filterOptions: filterOptions ?? undefined,
+          groupName: 'managed',
+          name: currentConfigurationData.alertName,
+          phoneNumber: finalPhoneNumber,
+          sourceId: source?.id ?? '',
+          targetGroupName: 'Default',
+          telegramId: finalTelegramId,
+        });
+
+        newResults[currentConfigurationData.alertName] = alert;
+      }
+    } else {
+      const alerts = data.alerts;
+
+      for (let i = 0; i < alerts.length; i++) {
+        const existingAlert = alerts[i];
+        console.log('existing alert', existingAlert);
         if (
-          source === undefined ||
-          source.id === null ||
-          filter === undefined ||
-          filter.id === null
+          existingAlert !== undefined &&
```

Comment:
it gets funky here. i create the alerts for configurations but if not, i attempt to update all existing alerts